README.md

```diff
@@ -165,3 +165,24 @@ yarn lint
 ```
 
 Checks if the code conforms the linting standards.
+
+## Deploying to the development AWS account
+
+We can deploy the app to our development environment for pre-merge testing.
+Only one branch can be deployed at a time because registering an OIDC client with Auth is a manual process at the moment.
+Before deploying, check with the team to see if anyone else is using it.
```

Comment:
Can we specify the channel we should use for notifications - e.g #govuk-accounts-tech

Reply:
Good idea